src/api/admin/types.ts

```diff
@@ -120,22 +121,22 @@ export interface ClonedCell {
   enabled: boolean;
 }
 
-/**
- * @public
- */
-export enum CellType {
```

Comment:
I prefer to keep this. It prevents errors when mistyping the expected type.

Reply:
Makes sense to me, yes. Added it back and updated the checks accordingly.